sqlstore/src/test/java/uk/ac/bbsrc/tgac/miso/sqlstore/AllTestsSuite.java

```diff
@@ -44,7 +44,7 @@
     SQLProjectDAOTest.class, SQLNoteDAOTest.class, SQLSequencerReferenceDAOTest.class, SQLRunQCDAOTest.class, SQLEmPCRDAOTest.class,
     SQLSecurityDAOTest.class, SQLSequencerPartitionContainerDAOTest.class, SQLBoxDAOTest.class, SQLSequencerServiceRecordDAOTest.class,
     SQLExperimentDAOTest.class, SQLPlatformDAOTest.class, SQLSequencerServiceRecordDAOTest.class, SQLPoolDAOTest.class,
-    SQLLibraryDAOTest.class, SQLLibraryDilutionDAOTest.class, SQLChangeLogDAOTest.class, SQLPrintJobDAOTest.class, SQLSecurityProfileDAOTest.class })
+    SQLLibraryDAOTest.class, SQLLibraryDilutionDAOTest.class, SQLChangeLogDAOTest.class, SQLPrintJobDAOTest.class, SQLSecurityProfileDAOTest.class, SQLPrintJobDAOTest.class, SQLPrintServiceDAOTest.class })
```

Comment:
`SQLPrintJobDAOTest.class` is already in this file, two items before the new addition.